solidity/ecdsa/test/WalletRegistry.Wallets.test.ts

```diff
@@ -363,4 +367,123 @@ describe("WalletRegistry - Wallets", async () => {
       )
     })
   })
+
+  describe("isWalletMember", () => {
+    let walletID: string
+    let walletMembersIDs: number[]
+    let walletMembersAddresses: string[]
+
+    before("create a wallet", async () => {
+      await createSnapshot()
+
+      let members: Operator[]
+      ;({ walletID, members } = await createNewWallet(
+        walletRegistry,
+        walletOwner.wallet
+      ))
+
+      walletMembersIDs = members.map((member) => member.id)
+      walletMembersAddresses = members.map((member) => member.signer.address)
+    })
+
+    after(async () => {
+      await restoreSnapshot()
+    })
+
+    context(
+      "when the given operator address is an actual sortition pool operator",
+      () => {
+        context("when the passed wallet members identifiers are valid", () => {
+          context("when the wallet member index is in correct range", () => {
+            context(
+              "when the given operator is the member of the wallet signing group at given position",
```

Comment:
*the given

Reply:
See https://github.com/keep-network/keep-core/pull/2925/commits/9904a69bcf49963814cde94b91042423819fd333